app/controllers/shipit/ccmenu_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Shipit
+  class CcmenuController < ActionController::Base
+    before_action :verify_signature, :check_permission
+
+    class NoDeploy
+      def id
+        0
+      end
+
+      def ended_at
+        DateTime.now.utc
```

Comment:
`DateTime` is rarely used in Ruby, it's almost deprecated. `Time.now.utc` will do just have well.